web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -233,3 +235,32 @@ function joinup_core_form_alter(&$form, FormStateInterface $form_state, $form_id
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ *
+ * Shows an information message to users who have access to the draft version
```

Comment:
The problem of using `hook_entity_view()` is that if we render the draft graph version somewhere else, we will get the message there too. Shouldn't we show this message only on the canonical route (as we said in the ticket)?